README.md

```diff
@@ -45,110 +45,9 @@ For convenience, you can use the provided script to generate a set of unique sel
-| SSH (22)        | TCP      |         22 | 0.0.0.0/0 | SSH            |
-| Custom TCP Rule | TCP      |      30399 | 0.0.0.0/0 | swarm TCP      |
-| Custom UDP Rule | UDP      |      30399 | 0.0.0.0/0 | swarm UDP      |
-| Custom TCP Rule | TCP      |       5000 | 0.0.0.0/0 | Onyx interface |
-| Custom TCP Rule | TCP      |       5002 | 0.0.0.0/0 | cert endpoint  |
-
-**Outbound**
-
-| Type        | Protocol | Port Range | Source    | Description |
-| ---         | ---      | ---        | ---       | ---         |
-| ALL Traffic | ALL      | ALL        | 0.0.0.0/0 | ALL Traffic |
-
-#### 2. Create a subnet for your Onyx server
-
-Go to [subnet management](https://eu-west-1.console.aws.amazon.com/vpc/home?region=eu-west-1#subnets:)
-in the AWS dashboard and create a new subnet in your VPC. Make sure it's within
-the vpc CIDR range. For example if the VPC CIDR is `10.0.0.0/16`, the sg
-IPv4 CIDR block can be `10.0.1.0/24`.
-
-#### 3. Create an SSH key
-You're going to use it to connect to the Onyx server EC2 node.
-
-```bash
-$ mkdir ~/ssh
-$ aws ec2 create-key-pair --key-name my_key --output text --query KeyMaterial > ~/ssh/my_key.pem
-$ chmod 400 ~/ssh/my_key.pem
-```
-
-#### 4. Launch the Onyx server
-
-Make sure you have [AWS CLI](https://aws.amazon.com/cli/) installed and configured.
-In terminal, run the following command:
-
-Find your security group and subnet ids, which should be something like `sg-XXXXXXXX`
-and `subnet-XXXXXXXX`, respectively. Use them in the following command
-
-```bash
-$ aws ec2 run-instances \
-    --image-id ami-1a066763 \
-    --instance-type t2.micro \
-    --key-name my_key \
-    --security-group-ids <SG ID HERE> \
-    --subnet-id <SUBNET ID HERE> \
-    --tag-specification "ResourceType=instance,Tags=[{Key=Name,Value=my_onyx_node}]" \
-    --associate-public-ip-address
-```
-
-This will launch your personal Onyx server. It will generate an
-account for you.
-
-#### 5. Fetch the certificates from the node
-
-In order to connect to the server, the client will need to use the right
-certificates - otherwise the connection will be rejected. They are generated on
-the server and you need to fetch them first.
-
-Find the public IP of the node you created
-[here](https://eu-west-1.console.aws.amazon.com/ec2/v2/home?region=eu-west-1#Instances:sort=instanceId)
-and copy the relevant files from it:
-
-```bash
-$ scp -i ~/ssh/my_key.pem ubuntu@<NODE PUBLIC IP HERE>:"~/certs/ca-crt.pem ~/certs/client-crt.pem ~/certs/client-key.pem" .
-```
-
-#### 6. Connect to your Onyx server
-
-Launch Onyx and as the `Onyx server websocket url` use
-`wss://<NODE PUBLIC IP HERE>:5000/graphql`. When prompted for the certificates
-use the one you downloaded in the previous step.
-
-You're connected!
-
 ### Development
 
-To build local version run `yarn start`. Afterwards you can start the built server
+To build local version run `yarn`. Afterwards you can start the built server
```

Comment:
`yarn start` is needed to build, simply running `yarn` only installs the dependencies.